script/setup

```diff
@@ -1,39 +1,5 @@
 #!/bin/sh
-set -e
-export PATH="$(dirname "$0")/../bin:$PATH"  # Use binstubs
+echo "DEPRECATED: This script has moved to bin/setup and no longer drops the database."
+echo "Running bin/setup"
+exec "$(dirname "$0")/../bin/setup"
```

Comment:
I wasn't sure about offering a message like this to help direct to bin/setup vs deleting script/setup entirely. Given the behavior was duplicated for the last 3+ years, I figure leaving a helpful message here is no big deal.

Reply:
If the file is useless and is not part of the docs anymore, I think we can just remove it. There is no need for deprecation in here.